README.md

```diff
@@ -622,7 +622,7 @@ Other Style Guides
     });
     ```
 
-  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise use a `return` statement.
+  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise add the parentheses, braces, and use a `return` statement.
```

Comment:
> explicit about whether the parens around a single arrow function argument were required

I guess that’s stated in [8.4](https://github.com/airbnb/javascript/tree/60480c05ea6c2d0f172302f845c39f5f62accb8d#8.4). The rule originally came from you guys – I just [bumped the number](https://github.com/airbnb/javascript/pull/472#discussion_r36888047).

> and […] that you must always use parens around arrow function args that have a body with braces

Yeah, I noticed that intention in the original text – and I did my best to make it explicit through “add *the parentheses*, braces, and use a `return` statement”. Originally it was also formulated this way.